### ISSUE TITLE
Added shapes don't show in saved widget state

### ISSUE THREAD

**Issue by frannerin** [nglviewer/nglview#1024]
I have a Python package that uses `nglview` to visualize elements (cylinders linking protein residues or spheres as residues) on protein structures. I am using `nbsphinx` to render a tutorial notebook (after running it locally and saving it with the 'Save widget state automatically' option on) into html to show in [my documentation](https://alloviz.readthedocs.io/en/latest/example.html). However, all the shapes (cylinders and spheres) don't show in the rendered html, nor even after saving, closing and reopening the notebook with Jupyter. I don't think https://github.com/nglviewer/nglview/issues/948#issuecomment-898121063 works for this case and the `write_html` function throws an error: `TypeError: Object of type float32 is not JSON serializable`.

Shapes seem to be in [the saved .ipynb file](https://raw.githubusercontent.com/frannerin/AlloViz/main/example.ipynb) (e.g., searching for "cylinder").

And, for example, this is what is supposed to look like in one of the cases:
![image](https://user-images.githubusercontent.com/74412019/174265781-e6ddc4df-e75f-4abc-8ee6-805cb6739d2f.png)

**Comment by hainm** [comment]
Dear @frannerin 
I can't reproduce your issue. By the way, I am using macos with latest nglview. Cheers.

<img width="1159" alt="image" src="https://user-images.githubusercontent.com/4451957/174463857-a37ff0c5-a3c1-44dd-87e6-dd5e7f2c7b2c.png">

<img width="613" alt="image" src="https://user-images.githubusercontent.com/4451957/174463866-a7d88a12-79ac-427e-a8e7-3ff97dd5e846.png">

**Comment by frannerin** [comment]
Sorry for the reproducibility issue! In the end I solved it by running the notebook and saving and closing, opening it back up (without being able to see the cylinders/shapes), saving and closing it again (Jupyter detected changes even if I had not done anything), and when I opened it back up or pushed it to build [documentation](https://alloviz.readthedocs.io/en/latest/example.html) the shapes were there. Thanks for your time!
